assets/src/edit-story/elements/image/display.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+/**
+ * Internal dependencies
+ */
+import { PAGE_WIDTH, PAGE_HEIGHT } from '../../constants';
+import { ElementWithPosition, ElementWithSize } from '../shared';
+import { getImgProps, ImageWithScale } from './util';
+
+const Element = styled.div`
+	${ ElementWithPosition }
+	${ ElementWithSize }
+	overflow: hidden;
+`;
+
+const Img = styled.img`
+	position: relative;
+	${ ImageWithScale }
+`;
+
+function ImageDisplay( { src, origRatio, width, height, x, y, scale, offsetX, offsetY } ) {
+	const actualRatio = width / height * PAGE_WIDTH / PAGE_HEIGHT;
```

Comment:
How does this formula work? Why is page's ratio necessary for an image?

Reply:
`width` and `height` are in percent if the actual page size. Thus, a size of 20x20 doesn't mean a square viewport, rather a viewport that is 20% of the page width and 20% of the page height. To figure out the actual viewport ratio we have to multiply it with the actual page ratio. With a static page ratio of 775/434 a 20x20 viewport image will have an actual ratio of about `1.79`.

Reply:
A comment above explaining this might be useful here.

Reply:
[Added comment](https://github.com/ampproject/amp-wp/pull/3804/files#diff-3351695c7709cca397b8b3229f4a4a94R27-R28).